src/System.Management.Automation/engine/CommonCommandParameters.cs

```diff
@@ -123,6 +123,27 @@ public ActionPreference InformationAction
             set { _commandRuntime.InformationPreference = value; }
         }
 
+        /// <summary>
+        /// Gets or sets the value of the ProgressAction parameter for the cmdlet.
+        /// </summary>
+        /// <remarks>
+        /// This parameter tells the command what to do when an progress record occurs.
+        /// </remarks>
+        /// <!--
+        /// NOTE: The "proga" alias name does not follow the same alias naming convention used
+        /// with other common parameter aliases that control stream functionality; however,
+        /// "pa" was already taken as a parameter alias in other commands when this parameter
+        /// was added to PowerShell, so "proga" was chosen instead.
+        /// -->
+        [Parameter]
+        [Alias("proga")]
```

Comment:
No doubt we'll get claims as to why it's not `pa'. We should immediately accept `pa` as a breaking change or remove this alias altogether - in scripts it is evil, but in the interactive session IntelliSense works fine.

Reply:
I originally had it as `pa`, but even some tests broke because `pa` was already in use.  I think it's too likely someone else is also using `pa` as an alias.  The alternative would be not to have an alias at all (we can always add it later).